docs/guide-send-tx-bundle.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Sending Tx and Bundles
```

Comment:
Using an acronym like "tx" in the title looks odd to me. What about one of these
- "How to use Flashbots"
- "Submitting to Flashbots"
- "User guide"

Reply:
How about searcher user guide?
Submitting to Flashbots might draw un-intended attention of people who want to submit blocks